core/src/main/java/com/linecorp/armeria/internal/DefaultValue.java

```diff
@@ -14,22 +14,25 @@
  * under the License.
  */
 
-package com.linecorp.armeria.server.http.dynamic;
+package com.linecorp.armeria.internal;
 
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
+import com.linecorp.armeria.server.http.dynamic.Param;
 
 /**
- * Annotation for mapping dynamic web requests onto specific parameter.
+ * Holds the default value used in annotation attributes.
  */
-@Retention(RetentionPolicy.RUNTIME)
-@Target(ElementType.PARAMETER)
-public @interface PathParam {
+public final class DefaultValue {
 
     /**
-     * Variable name in {@link Path}.
+     * String constant defining a value for unspecified by a user.
+     *
+     * @see Param#defaultValue()
      */
-    String value();
+    public static final String UNSPECIFIED = "\n\t\t\n\t\t\n\000\001\002\n\t\t\t\t\n";
```

Comment:
I referred to [ValueConstants.DEFAULT_NONE of Spring framework](https://github.com/spring-projects/spring-framework/blob/master/spring-web/src/main/java/org/springframework/web/bind/annotation/ValueConstants.java). 

I should avoid unicode character literals on a string constant, so I changed the unicode character to octal character.